PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -150,9 +150,6 @@
          puId106XUL17Disc = cms.InputTag('pileupJetId106XUL17:fullDiscriminant'),
          puId106XUL18Disc = cms.InputTag('pileupJetId106XUL18:fullDiscriminant'),
          chFPV0EF = cms.InputTag("jercVars:chargedFromPV0EnergyFraction"),
-         chFPV1EF = cms.InputTag("jercVars:chargedFromPV1EnergyFraction"),
-         chFPV2EF = cms.InputTag("jercVars:chargedFromPV2EnergyFraction"),
-         chFPV3EF = cms.InputTag("jercVars:chargedFromPV3EnergyFraction"),
```

Comment:
These variable need to be removed from the DQM as well

Reply:
Apologies. Will remove them from the DQM.